src/memory.asm

```diff
@@ -1589,12 +1592,21 @@ wD8DA:
 wDualCharacterScratch:
     ds 3                                               ;; d8db
 
-wJoypadInput:
-    ds 1                                               ;; d8de
+; 2-byte pairs of all 80 metatiles, pulled from the metatile data
+; and cached for faster access
+wMetatileAttributeCache:
+    ds 160
+
+; Used as scratch space for the spawnNpcsFromTable routine in bank03
+; First byte holds number of options (up to 165)
+; Subsequent 2-byte pairs provide a spawn location option (x followed by y).
+; This space can be reused for scratch space in its entirety outside of the spawnNpcsFromTable function.
+wSpawnPlacementScratch:
+    ds 331
 
 ; Free WRAM space.
 wFree:
-    ds 1825                                            ;; d8df
+    ds 1335                                            ;; d8df
```

Comment:
Please correct or remove the positional comment.

Reply:
Fixed